tests/py/test_billing_payday.py

```diff
@@ -501,8 +432,8 @@ def test_process_draws(self):
         with self.db.get_cursor() as cursor:
             payday.prepare(cursor)
             payday.process_payment_instructions(cursor)
-            payday.transfer_takes(cursor, payday.ts_start)
-            payday.process_draws(cursor)
+            payday.process_remainder(cursor)
+            payday.process_takes(cursor, payday.ts_start)
```

Comment:
Where the heck did this go? I no longer see the misordered calls in [payday.py](https://github.com/gratipay/gratipay.com/blob/51af22746bd7595170125f2f6298bc2755e0fa1e/gratipay/billing/payday.py#L156-L157) at 51af22746bd7595170125f2f6298bc2755e0fa1e, the current `head`, [nor in](https://github.com/gratipay/gratipay.com/blob/ee5bd7a586e89beeb888ce655381dac30f5823d7/gratipay/billing/payday.py#L156-L157) at ee5bd7a586e89beeb888ce655381dac30f5823d7, which was `head` when this comment was made. 😳